### PR TITLE
Fix: UIElementExtensions.Elevation breaks VS designer #5513

### LINKED ISSUE #5513: `UIElementExtensions.Elevation` breaks VS designer
## Current behavior
When setting `UIElementExtensions.Elevation` attached property for an element, the Visual Studio designer shows an error, but it compiles fine and works fine in a running app.

image

## Expected behavior
The VS designer must show the rendered XAML.

## How to reproduce it (as minimally and precisely as possible)
[App4.zip](https://github.com/unoplatform/uno/files/6167527/App4.zip)

Set `Elevation`, build the UWP project, see the error in the VS designer.

## Workaround
Set `UIElementExtensions.Elevation` via code-behind or don't use it at all.

## Environment

Nuget Package:

- [x] Uno.UI
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [x] Visual Studio 2019 (version: 16.9.2), Version 16.10.0 Preview 1.0
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

## Anything else we need to know?

To fix this, just remove `this` in `this UIElement element`. VS doesn't like these are extension methods.

https://github.com/unoplatform/uno/blob/a6bb7ecaceb1908172a896733c74871ca261a799/src/Uno.UI.Toolkit/UIElementExtensions.cs#L30

https://github.com/unoplatform/uno/blob/a6bb7ecaceb1908172a896733c74871ca261a799/src/Uno.UI.Toolkit/UIElementExtensions.cs#L35

### PR DESCRIPTION
**Pull request by artemious7**
GitHub Issue (If applicable): fixes #5513

## PR Type

What kind of change does this PR introduce?
Bugfix

## What is the current behavior?
VS designer breaks

## What is the new behavior?
VS designer fixed

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Toolkit/UIElementExtensions.cs [hunk 1]
```diff
@@ -27,12 +27,12 @@ public static class UIElementExtensions
 	{
 		#region Elevation
 
-		public static void SetElevation(this UIElement element, double elevation)
+		public static void SetElevation(UIElement element, double elevation)
```
Comment (jeromelaban) [inline src/Uno.UI.Toolkit/UIElementExtensions.cs:30]: This is unfortunately a breaking change. @carldebilly do you know if the extension method syntax is actually used somewhere ?

Comment (artemious7) [reply src/Uno.UI.Toolkit/UIElementExtensions.cs:30]: Oh, ok, sorry. Then how about creating a different class with these same extension methods that would set the attached property, while making the methods non-extension methods? It wouldn't break the callers who use it as an extension method

Comment (MartinZikmund) [reply src/Uno.UI.Toolkit/UIElementExtensions.cs:30]: @artemious7 Is it still the case that this causes issues for the designer?

Comment (Youssef1313) [reply src/Uno.UI.Toolkit/UIElementExtensions.cs:30]: We definitely need to re-evaluate whether this breaking change is needed. See https://github.com/unoplatform/uno/issues/5513#issuecomment-1532694800

**Comment by jeromelaban** [comment]
Thanks for your contribution. Since the designer is not supported in VS for WinUI at this time, we'll report the change to a later date.

**Pull request closed without merging**